doc/testing-kind.md

```diff
@@ -25,5 +62,12 @@ $ sudo make test-e2e-k8s
 $ ./hack/teardown-e2e-kind-cluster.sh
 ```
 
+#### Cleaning up the `system-service` service files
+```
+$ sudo rm -f /etc/systemd/system/vf-switcher.service
+$ rm -f /etc/vf-switcher/vf-switcher.yaml
```

Comment:
this is deleted in teardown-e2e-kind-cluster.sh so i dont think you need it here

Reply:
removed